internal/config/defs.go

```diff
@@ -1,8 +1,19 @@
 package config
 
-import "time"
+import (
+	"os"
+	"time"
+)
 
 const (
 	// DefaultTimeout is the default TURN credential timeout. Default is one day, in seconds.
 	DefaultTimeout = 24 * time.Hour
 )
+
+var PublicIp string
```

Comment:
Can we rename this to `PublicAddr`? The reason is that we want the user-enforced public address to come both as a regular IP both and a domain name, so perhaps the new name connotes this intention better.

Reply:
Done, thanks